frontend/src/components/NamespaceDropdown.tsx

```diff
@@ -193,27 +193,29 @@ class NamespaceDropdownComponent extends React.PureComponent<NamespaceDropdownPr
     );
   }
 
-  private getBody() {
+  private getBody(): JSX.Element {
     if (this.props.namespaces.length > 0) {
       const selectedMap = this.state.selectedNamespaces.reduce((map, namespace) => {
         map[namespace.name] = namespace.name;
         return map;
       }, {});
-      const namespaces = this.filtered().map((namespace: Namespace) => (
-        <div
-          className={optionStyle}
-          id={`namespace-list-item[${namespace.name}]`}
-          key={`namespace-list-item[${namespace.name}]`}
-        >
-          <input
-            type="checkbox"
-            value={namespace.name}
-            checked={!!selectedMap[namespace.name]}
-            onChange={this.onNamespaceToggled}
-          />
-          <span className={optionLabelStyle}>{namespace.name}</span>
-        </div>
-      ));
+      const namespaces = this.filtered()
+        .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
This line is the only actual thing that changed with this PR. The rest was just mandated by the new linter rules.